cves/CVE-2017-9805.yml

```diff
@@ -178,38 +193,48 @@ lessons:
     If you think of another lesson we covered in class that applies here, feel
     free to give it a small name and add one in the same format as these.
   defense_in_depth:
-    applies:
+    applies: false
     note:
   least_privilege:
-    applies:
+    applies: false
     note:
   frameworks_are_optional:
-    applies:
+    applies: false
     note:
   native_wrappers:
-    applies:
+    applies: false
     note:
   distrust_input:
-    applies:
-    note:
+    applies: true
+    note: |
+      The input was implicitly being trusted by the deserialization code, it
+      wasn't validating the input in any way, which it would have if it was
+      distrustful of the input.
```

Comment:
Breakdown the sentence and make it more straightforward. It is a little confusing what you are trying to say